### PR TITLE
Shipping with CSS templates out of the box

### PR DESCRIPTION
**Pull request by mistercrunch**
@williaster 
Also the modal becomes opacity 50% while editing css and moving the mouse out of the modal.

### CODE REVIEW

File: panoramix/assets/visualizations/pivot_table.css [hunk 1]
```diff
@@ -2,7 +2,12 @@
     overflow: auto !important;
 }
 
-.widget.pivot_table td,th {
-    padding: 1px 5px;
-    font-size: small;
+.table tr>th {
```
Comment (williaster) [inline panoramix/assets/visualizations/pivot_table.css]: It's generally really bad to use the `!important` flag

File: panoramix/data/__init__.py [hunk 2]
```diff
@@ -300,6 +301,105 @@ def load_birth_names():
...
+        font-size: 12px;
+        font-family: inherit;
+    }
+    body{
+        background: #000;
+        font-family: Courier, Monaco, monospace;;
+    }
+    /*
+    var bnbColors = [
+        //rausch    hackb      kazan      babu      lima        beach     barol
```
Comment (williaster) [inline panoramix/data/__init__.py]: I don't think `barol` is on of our colors `tirol`?

**Comment by williaster** [comment]
@mistercrunch this seems okay overall to me, but how are you thinking about this in relation to the overall bootstrap theme? Seems like we can ship that for devs that want to customize bootstrap vars, and use this as an easier interface for updating styles int he browser itself?

also what library is used to check the .css syntax in the modal? it was making suggestions for me like "this rule is too specific" that were actually _wrong_!

**Comment by mistercrunch** [comment]
Oh, so the wrong suggestion would be coming from `ace` I guess, we can probably shut this feature down or help fixing ace.

The theming is to allow people to customize the look and feel of individual dashboards by applying templates and modifying them. That css lives on top of the bootstrap theme and only affects individual dashboards. Live updating is a cool feature I think that makes iterating on designs quick.

File: panoramix/assets/visualizations/pivot_table.css (hunk 1 above)
Comment (mistercrunch) [inline panoramix/assets/visualizations/pivot_table.css]: I'm not sure how else to compete with the very high selectivity css in the bootstrap/dataTables css...

Comment (williaster) [inline panoramix/assets/visualizations/pivot_table.css]: that can be tricky sometimes, okay.

**Comment by williaster** [comment]
:+1:

**Pull request merged**